packages/wrangler/src/dev/start-server.ts

```diff
@@ -135,6 +135,9 @@ export async function startDevServer(
 			format: props.entry.format,
 			moduleRoot: props.entry.moduleRoot,
 		},
+		legacy: {
+			enablePagesAssetsServiceBinding: props.enablePagesAssetsServiceBinding,
+		},
```

Comment:
I think this is OK to put in legacy for now. But we might need to revisit this when we get Assets for Workers up and running.

Reply:
I've updated this PR to pass in a fetcher binding and therefore do not need to add this extra option to the API and support it forever